CMSgov/design-system
# [NO-TICKET] Rename SlidingPanel to Drawer 👖

## Summary
No ticket.

Renamed SlidingPanel to Drawer as it makes more sense when HelpDrawer exists in the system.

## How to test
Visit the [Drawer documentation](http://design-system-demo.s3-website-us-east-1.amazonaws.com/rename-slidingpanel/components/drawer) and make sure everything looks good and makes sense.
